OsmAnd/res/values/strings.xml

```diff
@@ -14,26 +14,26 @@
     <string name="gps_filter_precision">GPS precision</string>
     <string name="shared_string_precision">Precision</string>
     <string name="shared_string_statistics">Statistics</string>
-    <string name="gps_filter_hdop_desc">Set the maximum accepted value for HDOP.\nPoints with higher value will be hidden.</string>
-    <string name="gps_filter_speed_altitude_desc">Only track points matching the set interval will appear on the chart and map, the rest will be hidden.</string>
-    <string name="gps_filter_smoothing_desc">Set the threshold distance between points.\nTrack points not at least this distance away from the last visible point will be hidden.\nBe aware that high thresholds may oversimplify the track geometry.</string>
+    <string name="gps_filter_hdop_desc">Set the max. value for HDOP.\nPoints farther away are hidden.</string>
```

Comment:
Wrong meaning: ,,, neither is this about setting an hdop, nor has this anything to do with "points too far away"...